lib/ember-test-helpers/build-registry.js

```diff
@@ -47,6 +47,14 @@ export default function(resolver) {
     registry = new Ember.Registry({
       fallback: fallbackRegistry
     });
+
+    // these properties are set on the fallback registry by `buildRegistry`
+    // and on the primary registry within the ApplicationInstance constructor
+    // but we need to manually recreate them since ApplicationInstance's are not
+    // exposed externally
+    registry.normalizeFullName = fallbackRegistry.normalizeFullName;
+    registry.makeToString = fallbackRegistry.makeToString;
```

Comment:
For consistency, we should probably also set `registry.describe = fallbackRegistry.describe;` 

See https://github.com/emberjs/ember.js/blob/master/packages/ember-application/lib/system/application.js#L1347-L1349